modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -169,23 +177,130 @@ Where the [n] is replaced by the value for each of the date and time elements th
 - `M` is the minute designator (e.g. `|PT1M|`)
 - `S` is the second designator (e.g. `|PT1S|`)
 
-.Transform
+`DatePeriod` is useful for date manipulation, such as addition or subtraction. +
+The following example shows how to subtract one year from a date using `DatePeriod`. The return type is a new `Date`:
+
+.Source
+[source,dataweave,linenums]
+----
+output application/dw
+---
+|2003-10-01| - |P1Y|
+----
+
+.Output
+[source,dataweave,linenums]
+----
+|2002-10-01|
+----
+
+`Duration` is useful as a result of a `Date` to `Date` subtraction so the amount of time between those two dates can be inferred. +
```

Comment:
```suggestion
`Duration` is useful when you want to subtract one `Date` value from another to determine the amount of time between the two dates. +
```

Reply:
I wonder if the usefulness of each method should accompany its relative definition, earlier?